oculus_ros2/include/oculus_ros2/oculus_viewer_node.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 Forssea Robotics
+// All rights reserved.
+//
+// Unauthorized copying of this code base via any medium is strictly prohibited.
+// Proprietary and confidential.
+
+#ifndef OCULUS_ROS2__OCULUS_VIEWER_NODE_HPP_
+#define OCULUS_ROS2__OCULUS_VIEWER_NODE_HPP_
+
```

Comment:
@AugusteBourgois I still have `src/oculus_sonar/oculus_ros2/include/oculus_ros2/oculus_viewer_node.hpp:41:  Found C system header after other header. Should be: oculus_viewer_node.h, c system, c++ system, other.  [build/include_order] [4]` when I am runing pre commit